src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -745,7 +745,8 @@ else if (AbstractCheck.class.isAssignableFrom(clss)) {
             Arrays.sort(requiredTokens);
 
             if (!Arrays.equals(acceptableTokens, defaultTokens)
-                    || !Arrays.equals(acceptableTokens, requiredTokens)) {
+                    || !Arrays.equals(acceptableTokens, requiredTokens)
+                    || "DescendantToken".equals(sectionName)) {
```

Comment:
` || acceptableTokens.size() == TOKEN_TYPE_TOTAL_COUNT)`

but why we need this condition, `Arrays.equals` will never be true.
Please remove.

Reply:
Why not? both acceptableTokens and defaultTokens is empty for DescendantTokenCheck, it is an exception for all the cases.

Reply:
https://github.com/checkstyle/checkstyle/blob/3686c0f493e37831d1450b6ac608b9734fb7efa0/src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java#L477

acceptableTokens is not empty.

Reply:
fixed.